src/test/java/scooterLoginCourierWithoutPasswordTest.java

```diff
@@ -0,0 +1,56 @@
+// импортируем RestAssured
+// импортируем Response
+
+import io.qameta.allure.Description;
+import io.qameta.allure.junit4.DisplayName;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.Before;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class scooterLoginCourierWithoutPasswordTest {
+
+    private CourierClient courierClient;
+
+    @Before
+    public void setUp() {
+        courierClient = new CourierClient();
+    }
+
+    @Test
+    @Description("Логин курьера без пароля")
+    @DisplayName("Логин курьера без подстановки password")
+    public void loginCourierWithoutPasswordTest(){
```

Comment:
Твой тест падает с очень не хорошей ошибкой:
_Cannot invoke the path method because no content-type was present in the response and no default parser has been set.
You can specify a default parser using e.g.:
RestAssured.defaultParser = Parser.JSON;_
Со стороны это выглядит как косяк тестов, а не косяк сервиса. А это не так. Такая ситуация происходит из-за того, что наш сервер виснет не отдает тело ответа, но зато он отдает статус код. Если проверка на статус код делалась бы раньше проверки body, то тест бы падал с красивой ошибкой статус кода и вопросов бы не было. 
Поэтому я всегда рекомендую первым делом проверять статус код, а потом уже все остальное.